### ISSUE TITLE
Timeout error on elasticsearch bulk operation

### ISSUE THREAD

**Issue by vkobel** [vkobel/ethdrain#6]
```
 Traceback (most recent call last):    
   File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker    
     result = (True, func(*args, **kwds))    
   File "/usr/lib/python3.5/multiprocessing/pool.py", line 44, in mapstar    
     return list(map(*args))    
   File "./ethdrain.py", line 123, in setup_process    
     helpers.bulk(elasticsearch, out_actions)    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/helpers/__init__.py", line 194, in bulk    
     for ok, item in streaming_bulk(client, actions, **kwargs):    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/helpers/__init__.py", line 162, in streaming_bulk    
     for result in _process_bulk_chunk(client, bulk_actions, raise_on_exception, raise_on_error, **kwargs):    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/helpers/__init__.py", line 91, in _process_bulk_chunk    
     raise e    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/helpers/__init__.py", line 87, in _process_bulk_chunk    
     resp = client.bulk('\n'.join(bulk_actions) + '\n', **kwargs)    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/client/utils.py", line 73, in _wrapped    
     return func(*args, params=params, **kwargs)    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/client/__init__.py", line 1167, in bulk    
     doc_type, '_bulk'), params=params, body=self._bulk_body(body))    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/transport.py", line 318, in perform_request    
     status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)    
   File "/usr/local/lib/python3.5/dist-packages/elasticsearch/connection/http_urllib3.py", line 122, in perform_request    
     raise ConnectionTimeout('TIMEOUT', str(e), e)    
 elasticsearch.exceptions.ConnectionTimeout: ConnectionTimeout caused by - ReadTimeoutError(HTTPConnectionPool(host='localhost', port=9200): Read timed out. (read
  timeout=10))    
 """    
     
 The above exception was the direct cause of the following exception:    
     
 Traceback (most recent call last):    
   File "./ethdrain.py", line 151, in <module>
     POOL.map(setup_process, CHUNKS)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 260, in map                                                                                                                                                                              
    return self._map_async(func, iterable, mapstar, chunksize).get()                            
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 608, in get                                                                                                                                                                              
    raise self._value                                                                              
elasticsearch.exceptions.ConnectionTimeout: ConnectionTimeout caused by - ReadTimeoutError(None: None)
```
helpers.bulk line seems to timeout in some cases

**Comment by vkobel** [comment]
Should probably:
* lower the CHUNK_SIZE to a more reasonable number (500 Blocks could be HUGE depending on how many tx there is in).
* increase the timeout value (default is 10 sec).

**Comment by vkobel** [comment]
fixed by fec31426f605eb75659df3de0e90c969f72a565b
